src/SprykerEco/Zed/Payone/Communication/Plugin/Log/PaymentLogReceiverPluginInterface.php

```diff
@@ -18,5 +19,5 @@ interface PaymentLogReceiverPluginInterface
      *
      * @return \Generated\Shared\Transfer\PayonePaymentLogCollectionTransfer
      */
-    public function getPaymentLogs(ObjectCollection $orders);
+    public function getPaymentLogs(ObjectCollection $orders): PayonePaymentLogCollectionTransfer;
```

Comment:
It's a part of public API, please remove added type hints.

Reply:
Fixed.